test/images/agnhost/VERSION

```diff
@@ -1 +1 @@
-2.8
+2.9
```

Comment:
maybe we should start just using the sha or something instead of a version number bound to have conflicts ...?

Reply:
discussing in #release-management

Reply:
this is probably a good idea to deal with in a follow up, additionally though auto-image pushing currently does not work, so I'll go look into that first...

Reply:
https://github.com/kubernetes/kubernetes/pull/87162